tests/test_bad_subprocess_use.py

```diff
@@ -33,15 +36,61 @@ def test_bad_subprocess_usage(self):
         result = linter.get_results()
         expected = [
             dlint.linters.base.Flake8Result(
-                lineno=4,
+                lineno=6,
+                col_offset=0,
+                message=dlint.linters.BadSubprocessUseLinter._error_tmpl
+            ),
+            dlint.linters.base.Flake8Result(
+                lineno=7,
+                col_offset=0,
+                message=dlint.linters.BadSubprocessUseLinter._error_tmpl
+            ),
+            dlint.linters.base.Flake8Result(
+                lineno=8,
                 col_offset=0,
                 message=dlint.linters.BadSubprocessUseLinter._error_tmpl
             ),
             dlint.linters.base.Flake8Result(
-                lineno=5,
+                lineno=9,
                 col_offset=0,
                 message=dlint.linters.BadSubprocessUseLinter._error_tmpl
             ),
+            dlint.linters.base.Flake8Result(
+                lineno=10,
+                col_offset=0,
+                message=dlint.linters.BadSubprocessUseLinter._error_tmpl
+            ),
+            dlint.linters.base.Flake8Result(
+                lineno=11,
+                col_offset=0,
+                message=dlint.linters.BadSubprocessUseLinter._error_tmpl
+            ),
+        ]
+
+        assert result == expected
+
+    def test_subprocess_usage(self):
+        python_node = self.get_ast_node(
+            """
+            import subprocess
+
+            command = "/bin/echo"
+
+            subprocess.call(command)
+            subprocess.call(shell=False)
+            subprocess.check_call(shell=False)
+            subprocess.check_output(shell=False)
+            subprocess.Popen(shell=False)
+            subprocess.run(command)
+            subprocess.run(shell=False)
+            """
+        )
+
+        linter = dlint.linters.BadSubprocessUseLinter()
+        linter.visit(python_node)
+
+        result = linter.get_results()
+        expected = [
```

Comment:
In this case I think it should be `expected = []` and `assert result == expected` since all `subprocess` calls are either implicitly or explicitly setting `shell=False`. Right?

Reply:
Much cleaner. 👍